src/components/NcListItem/NcListItem.vue

```diff
@@ -747,41 +732,54 @@ export default {
 // NcListItem
 .list-item {
 	box-sizing: border-box;
-	display: block;
+	display: flex;
 	position: relative;
 	flex: 0 0 auto;
 	justify-content: flex-start;
 	padding: 8px 10px;
+	// 4px padding for the focus-visible styles
+	margin: 4px;
```

Comment:
That offset a list items content at 4px from the left border, cropping them
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/93392545/139d435a-c4a5-42ed-b087-575b885bad90)

Maybe we should consider it like this, @susnux ?
```css
width: calc(100% - 8px);
```